wwwbase/scramble.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require_once("../phplib/util.php");
+setlocale(LC_ALL, "ro_RO.utf8");
+
+SmartyWrap::assign('page_title', 'Omleta Cuvintelor');
+SmartyWrap::addCss('scramble');
+SmartyWrap::addJs('scramble');
+SmartyWrap::addJs('gallery');
```

Comment:
Nu pot sa verific daca functioneaza cu noul jcanvas. Atunci cand caut tastiera primesc urmatoarea eroare:
 ftp_login(): Login authentication failed in /var/www/DEX/phplib/FtpUtil.php on line 11